docs/usage/actions/actions.rst

```diff
@@ -0,0 +1,109 @@
+Actions
```

Comment:
Common feedback on the "Usage" section.

"Usage section" duplicates "Api client" section.
For example, links:
https://hcloud-python.readthedocs.io/en/documentation/api.clients.actions.html
https://hcloud-python.readthedocs.io/en/documentation/usage/actions/actions.html#get-all-actions

contains the same information, except examples. I suggest moving examples to docstring of respective clients, and delete "Usage" section.

Reply:
Or we can drop the Usage section complete, because we have some samples already there (https://hcloud-python.readthedocs.io/en/documentation/samples.html) What do you think?

Reply:
Agree.